.github/workflows/primer-test.yaml

```diff
@@ -64,7 +64,7 @@ jobs:
           pip install -U -r requirements_test.txt
 
   pytest-primer-stdlib:
-    name: Run primer tests on stdlib Python ${{ matrix.python-version }} (Linux)
+    name: run on stdlib / ${{ matrix.python-version }} / Linux
```

Comment:
```suggestion
    name: primer / stdlib / ${{ matrix.python-version }} / Linux
```

Reply:
Primer is already included in the workflow name, see the names in the checks of this PR